depends/packages/boost.mk

```diff
@@ -1,8 +1,8 @@
 package=boost
-$(package)_version=1_73_0
-$(package)_download_path=https://boostorg.jfrog.io/artifactory/main/release/$(subst _,.,$($(package)_version))/source/
-$(package)_file_name=boost_$($(package)_version).tar.bz2
-$(package)_sha256_hash=4eb3b8d442b426dc35346235c8733b5ae35ba431690e38c6a8263dce9fcbb402
+$(package)_version=1.77.0
```

Comment:
package version format changes should be done via 24276 backport and not squashed silently into 24383